lib/omniauth/strategies/cronofy_base.rb

```diff
@@ -17,7 +17,7 @@ def data_center_env
 
       def data_center_url(type, value)
         case value.to_s
-        when 'de'
+        when 'au', 'de', 'ca', 'sg', 'uk'
```

Comment:
This might be closer to what we have in other SDKs and a bit more future proof:

```suggestion
        when 'us'
          "https://#{type}.cronofy.com"
        when String
          "https://#{type}-#{value}.cronofy.com
        end
```

Reply:
As we call `to_s` on value perhaps too accepting for the second case, `nil` should fall through to the default for example.